src/Jackett.Common/Indexers/BaseIndexer.cs

```diff
@@ -246,6 +246,20 @@ protected virtual bool IsValidRelease(ReleaseInfo release)
                 return false;
             }
 
+            if (release.Size == null || release.Size == 0.0)
+            {
+                logger.Warn("Invalid Release: '{0}' from indexer: {1}. No size provided.", release.Title, Name);
+
+                return false;
+            }
+
+            if (release.Category == null || !release.Category.Any())
+            {
+                logger.Warn("Invalid Release: '{0}' from indexer: {1}. No categories provided.", release.Title, Name);
```

Comment:
not as useful as the previous version, in that it does not cater for a new site category that the indexer caps currently does not have.